Update ANGLE to 2022-08-17 (92e4cd75785d257a5020b197ca382b06c32e9722)

kkinnunen-apple opened the pull request:
#### d4f6a25dd0aa29017f6c9fe3ee588d6f8afe92e5
<pre>
Update ANGLE to 2022-08-17 (92e4cd75785d257a5020b197ca382b06c32e9722)
<a href="https://bugs.webkit.org/show_bug.cgi?id=244030">https://bugs.webkit.org/show_bug.cgi?id=244030</a>
rdar://problem/98778814

Reviewed by Kenneth Russell

Contains hunk fixing iOS Simulator IsMetalRendererAvailable().

Contains upstream commits:
git log --oneline 7399fc7f6606f3b30de3f1113002a2c73d36968b..92e4cd75785d257a5020b197ca382b06c32e9722 --pretty=%h %s
92e4cd757 Roll SwiftShader from c61fc9aec162 to 5ac9bd16c803 (3 revisions)
213a1ee6e Vulkan: Enable VK_EXT_primitives_generated_query
a216cfe8b Adding angle_perftests test case names in logcat
7c66d77d6 Avoid logging None introduced in <a href="https://crrev.com/c/3831885">https://crrev.com/c/3831885</a>
c5efd2b5d Improve Ctrl-c handling.
e79c9cd63 Vulkan: Use VK_EXT_primitive_topology_list_restart
428b6788d Update CHROMIUM_copy_texture.txt to match the code
6ee6fac97 Roll SwiftShader from a6c3e363b7a9 to c61fc9aec162 (1 revision)
8878366fc Refactor to reduce main loop complexity.
ad6513fe3 Unconditionally enable EGL_ANGLE_iosurface_client_buffer on macOS
19eb658f6 Reland &quot;Infra: Enable Skia Gold tests on Pixel 6&quot;
0f5770dea Roll Chromium from 16d8164e92a5 to 754ede9fc072 (380 revisions)
2089943b9 Tests: Add Lumino City trace
b9fd47b8d Not use Android TLS slot for chromium
64c652d7f Use stdout instead of (default) stderr in logging.StreamHandler
72f03f793 Roll SwiftShader from 2438c1b18a5e to a6c3e363b7a9 (1 revision)
042fda3ca Roll Chromium from f904dc881cb8 to 16d8164e92a5 (630 revisions)
4c9411497 External Texture Support In MEC
02d79c32c Skip kotor trace on Swiftshader due to flakiness
44791792b Use angle::FlatUnorderedSet for ImageSibling::mSourcesOf
4cdabfb14 Roll VK-GL-CTS from eaf78cbfdaed to 5a93cd60f33b (4 revisions)
a9fac2491 Use FixedVector for Framebuffer::mDirtyColorAttachmentBindings
67e0c6fcc PRESUBMIT: Remove unnecessary escape of forward slash
1ca944482 Roll SwiftShader from 708ca9579181 to 2438c1b18a5e (2 revisions)
002b9544a Roll Chromium from e4653aba963d to f904dc881cb8 (487 revisions)
afdd51060 Use angle::FlatUnorderedMap in Display.cpp
2ebd51009 Add gl::Context as a parameter to Shader::resolveCompile
61a17ba9f Use angle::FlatUnorderedSet for ImageState::targets
583927912 Translator: Fix crashes with monomorphizing loops
4c6f7d8ec Vulkan: Never stage update for EGLImage
c9360ccbd Reflect TextureD3D label updates with storage object
4a636cdd4 Require all PLS formats to consume exactly 4 bytes of storage
badfeecde Vulkan: Destroy fb1 should not affect fb2 with same attachments
fa3d7d5cd Make PLS coherent on Vulkan
aef9603f1 Separate Struct declarations earlier
47d2df0ec Presubmit: Verify ANGLE_SH_VERSION update
4cab7f38e Fix EGL_ANGLE_program_cache_control for eglCreateContext
87ed2c9d1 TestSuiteTest: Generate more unique result file names.
26bb1d6e8 Roll vulkan-deps from c728ed48ce2f to d53c356ad7d9 (3 revisions)
58b4dbd8b Roll SwiftShader from aafa10869568 to 708ca9579181 (4 revisions)
6c9530690 Roll Chromium from d830698e47ae to e4653aba963d (476 revisions)
3d0faa02e Remove workaround affecting --list-tests --gtest_filter=X.
876da4ae9 Vulkan: Query support for VK_KHR_imageless_framebuffer
f7d4c6145 Vulkan: Use VK_EXT_primitives_generated_query
6d0452b73 Fix ANGLE_HAS_HISTOGRAMS following crrev.com/c/3817602
7c6a5459d Rename ShFragmentSynchronizationType::None -&gt; NoSynchronization
493b5aff7 Vulkan: Workaround ARM bug with stencil write mask
dc82ff15e Add config angle_maybe_has_histograms to angle_perftests_common.
17df2fd32 Roll vulkan-deps from fe14e7954b25 to c728ed48ce2f (8 revisions)
408c5a56b Mark FormatPrintTest as potentially uninstantiated.
ee0938386 Roll Chromium from 6306836d087b to d830698e47ae (506 revisions)
2c3513518 Vulkan: Don&apos;t break render pass on read-only buffer updates
1bb3685d0 Roll SwiftShader from 678ffefb1b3b to aafa10869568 (9 revisions)
8050079c1 Vulkan: Remove basicGLLineRasterization
52719c9f7 [fuchsia] Migrate angle_end2end_tests to component framework v2.
d8c6a2357 Roll vulkan-deps from 4b211a6a98e6 to fe14e7954b25 (6 revisions)
97b16799c Roll Chromium from 199441d23d93 to 6306836d087b (461 revisions)
6f80f0f03 Translator: Clean up the compile flag passing interface
8474a95e3 Translator: Don&apos;t validate structs on non-root nodes
7fe0694ce Vulkan: Use push constants for driver uniforms
c554b92b1 Apply memory qualifier decorations in SPIR-V
2cf9b1d63 Encapsulate prevTexture pointer in a class
52b18311b Use angle::HashSet to replace std::set in Display.h
f5de8e014 Vulkan: Disable texture upload if texture sharing
5b7b7f9b0 Document how to roll VMA for ANGLE
aaf293c89 Roll VK-GL-CTS from 0170b4a25e36 to eaf78cbfdaed (4 revisions)
2de8bb42a Converge run wrappers into angle_test_util.RunTestSuite
a2ad517f9 Replace std::vector with angle::FixedVector in FramebufferState.
22b90986c Roll vulkan-deps from fa6a3bcb6169 to 4b211a6a98e6 (8 revisions)
bf3955820 Roll Chromium from 5dde4351e359 to 199441d23d93 (45 revisions)
6491396c1 Use angle::FlatUnorderedMap as storage for AttributMap
ae0bb7416 Roll Chromium from 2bf3e35d7a4c to 5dde4351e359 (604 revisions)
6b248a688 Use angle::FastVector in ImageEGL::initialize to avoid heap allocation
0c8c4ca02 Remove not used mImageLastUseFences in ImageVk
0d5ba4dc4 Minor performance improvement
928c5016b Vulkan: Fix garbage collection vs outside-RP-only flush
d53086022 Test for ARM bug with dynamic stencil write mask
146329b81 run_perf_tests: Allow passing flags to the test.
9e3e20327 Add a note about &quot;internal swarming failure&quot;
123bd5422 Add Variable Initialization in Base Instance Tests
a46fdafd3 Roll vulkan-deps from 3a443029375b to fa6a3bcb6169 (5 revisions)
dbfab1595 Roll Chromium from d7560fcbdf63 to 2bf3e35d7a4c (416 revisions)
de5c6c791 Remove unused ANGLE histograms
6f158ca04 Roll vulkan-deps from 3ee9789f25ed to 3a443029375b (8 revisions)
861149c7a Make PLS coherent on desktop OpenGL
e6fcb2a9a Roll Chromium from 54b1e2bc5d71 to d7560fcbdf63 (301 revisions)
c0818a182 Protect ImageState::targets with a std::mutex
9f0cec145 dEQP: Fix X11 display config name.
c54ed7905 Get desktop GL conformance tests to build
448e2275f (Reland) [Metal] Disable ANGLE&apos;s Metal backend on older GPUs.
57ccd974b Make limit_max_texture_size_to_4096 webgl only
88b05d3c8 Use globals in android_helper to simplify flow.
553b1334e Vulkan: fix default msaa framebuffer resolve issue.
00a9787e3 Roll Chromium from fe6c14e1fc75 to 54b1e2bc5d71 (162 revisions)
8f14296cc Roll VMA to version 3.0.1 + Google patches
6bc106f74 Ignore all .gclient* files.
e88cab292 Revert &quot;Skip multi draw indirect tests on Linux/Intel/Vulkan.&quot;
f0d2dbec7 Retry flaky failures on SwiftShader *SAN builds.
33c09ba9b Fix local gold runs broken by previous refactoring.
25a5a42df Roll vulkan-deps from 9501e63393f3 to 3ee9789f25ed (3 revisions)
0b7e347a6 Roll Chromium from 0920d63e6fcc to fe6c14e1fc75 (234 revisions)
05d86e060 Add temporary logging of gold_session.Initialize stdout.
47684b26f Roll Chromium from d268b4c8a6b1 to 0920d63e6fcc (256 revisions)
351a2a6b6 Skip multi draw indirect tests on Linux/Intel/Vulkan.
19c6600c4 Refactoring: move a few common functions to angle_test_util.
13da8b973 Tests: Add Octopath Traveler trace
c0f2a5ac8 Skip TimerQueriesTest flakes on NV/Linux.
245198f1e Skip flaky tests on Linux/Vulkan.
f62b4b940 Skip flaky image load/store tests on Win/D3D11.
c29100a8f Skip flaky test on SwiftShader.
6e6f3e6da Convert GLenum to string for failed trace prereq message
1353f1191 Roll vulkan-deps from 4818e22b9347 to 9501e63393f3 (1 revision)
8eaa0ccfc Revert &quot;[Metal] Disable ANGLE&apos;s Metal backend on older GPUs.&quot;
0d0d9168c Roll SwiftShader from 67158b90f83c to 678ffefb1b3b (2 revisions)
60a02ca6e Roll Chromium from bfa66f9bea8c to d268b4c8a6b1 (202 revisions)
cc2095d25 [Metal] Disable ANGLE&apos;s Metal backend on older GPUs.
c517463ab Add NoUnclampedBlendColor limitation
81554b66a Vulkan: Reduce Framebuffer dirtyBits for swapchain only changes
49523e69d Roll Chromium from e810ac85e1c8 to bfa66f9bea8c (1114 revisions)
606a92395 Tests: Add Ni no Kuni trace
0b4ee2a8b Roll vulkan-deps from 73766817e984 to 4818e22b9347 (9 revisions)
6887c9505 Roll SwiftShader from 80aaa8a74b74 to 67158b90f83c (2 revisions)
dc77621bc Vulkan: Fix RefCountedDescriptorPoolHelper leak
d6acced1f Add flag --save-screenshots (implied by --screenshot-dir).

Canonical link: <a href="https://commits.webkit.org/253669@main">https://commits.webkit.org/253669@main</a>
</pre>

<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/08982a89fb55daa02e3ff405a56f39615012df25

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/86669 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/69/builds/30728 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/17602 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/95512 "Built successfully") | [✅ 🛠 🧪 win](https://ews-build.webkit.org/#/builders/10/builds/149243 "Built successfully and passed tests") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/64/builds/29113 "Built successfully") | [✅ 🛠 mac-debug](https://ews-build.webkit.org/#/builders/71/builds/25529 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/78850 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/90755 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/92285 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/68/builds/23516 "Passed tests") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/73591 "Built successfully") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/23584 "Passed tests") | 
| | [⏳ 🧪 api-ios](https://ews-build.webkit.org/#/builders/API-Tests-iOS-Simulator-EWS "Waiting to run tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/78868 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/66591 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/66/builds/26885 "Built successfully") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/12720 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/67/builds/26804 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/13735 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/2602 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/63/builds/28482 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/60/builds/36596 "Passed tests") | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/65/builds/28426 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/33014 "Passed tests") | | 
<!--EWS-Status-Bubble-End-->

kenrussell reviewed (approved):
Looks good to me. Let's keep ANGLE in WebKit up to date.

I reviewed everything in changes.diff and all the incoming changes in libANGLE/renderer/metal/ .

kkinnunen-apple committed:
Update ANGLE to 2022-08-17 (92e4cd75785d257a5020b197ca382b06c32e9722)
https://bugs.webkit.org/show_bug.cgi?id=244030
rdar://problem/98778814

Reviewed by Kenneth Russell

Contains hunk fixing iOS Simulator IsMetalRendererAvailable().

Contains upstream commits:
git log --oneline 7399fc7f6606f3b30de3f1113002a2c73d36968b..92e4cd75785d257a5020b197ca382b06c32e9722 --pretty=%h %s
92e4cd757 Roll SwiftShader from c61fc9aec162 to 5ac9bd16c803 (3 revisions)
213a1ee6e Vulkan: Enable VK_EXT_primitives_generated_query
a216cfe8b Adding angle_perftests test case names in logcat
7c66d77d6 Avoid logging None introduced in https://crrev.com/c/3831885
c5efd2b5d Improve Ctrl-c handling.
e79c9cd63 Vulkan: Use VK_EXT_primitive_topology_list_restart
428b6788d Update CHROMIUM_copy_texture.txt to match the code
6ee6fac97 Roll SwiftShader from a6c3e363b7a9 to c61fc9aec162 (1 revision)
8878366fc Refactor to reduce main loop complexity.
ad6513fe3 Unconditionally enable EGL_ANGLE_iosurface_client_buffer on macOS
19eb658f6 Reland "Infra: Enable Skia Gold tests on Pixel 6"
0f5770dea Roll Chromium from 16d8164e92a5 to 754ede9fc072 (380 revisions)
2089943b9 Tests: Add Lumino City trace
b9fd47b8d Not use Android TLS slot for chromium
64c652d7f Use stdout instead of (default) stderr in logging.StreamHandler
72f03f793 Roll SwiftShader from 2438c1b18a5e to a6c3e363b7a9 (1 revision)
042fda3ca Roll Chromium from f904dc881cb8 to 16d8164e92a5 (630 revisions)
4c9411497 External Texture Support In MEC
02d79c32c Skip kotor trace on Swiftshader due to flakiness
44791792b Use angle::FlatUnorderedSet for ImageSibling::mSourcesOf
4cdabfb14 Roll VK-GL-CTS from eaf78cbfdaed to 5a93cd60f33b (4 revisions)
a9fac2491 Use FixedVector for Framebuffer::mDirtyColorAttachmentBindings
67e0c6fcc PRESUBMIT: Remove unnecessary escape of forward slash
1ca944482 Roll SwiftShader from 708ca9579181 to 2438c1b18a5e (2 revisions)
002b9544a Roll Chromium from e4653aba963d to f904dc881cb8 (487 revisions)
afdd51060 Use angle::FlatUnorderedMap in Display.cpp
2ebd51009 Add gl::Context as a parameter to Shader::resolveCompile
61a17ba9f Use angle::FlatUnorderedSet for ImageState::targets
583927912 Translator: Fix crashes with monomorphizing loops
4c6f7d8ec Vulkan: Never stage update for EGLImage
c9360ccbd Reflect TextureD3D label updates with storage object
4a636cdd4 Require all PLS formats to consume exactly 4 bytes of storage
badfeecde Vulkan: Destroy fb1 should not affect fb2 with same attachments
fa3d7d5cd Make PLS coherent on Vulkan
aef9603f1 Separate Struct declarations earlier
47d2df0ec Presubmit: Verify ANGLE_SH_VERSION update
4cab7f38e Fix EGL_ANGLE_program_cache_control for eglCreateContext
87ed2c9d1 TestSuiteTest: Generate more unique result file names.
26bb1d6e8 Roll vulkan-deps from c728ed48ce2f to d53c356ad7d9 (3 revisions)
58b4dbd8b Roll SwiftShader from aafa10869568 to 708ca9579181 (4 revisions)
6c9530690 Roll Chromium from d830698e47ae to e4653aba963d (476 revisions)
3d0faa02e Remove workaround affecting --list-tests --gtest_filter=X.
876da4ae9 Vulkan: Query support for VK_KHR_imageless_framebuffer
f7d4c6145 Vulkan: Use VK_EXT_primitives_generated_query
6d0452b73 Fix ANGLE_HAS_HISTOGRAMS following crrev.com/c/3817602
7c6a5459d Rename ShFragmentSynchronizationType::None -> NoSynchronization
493b5aff7 Vulkan: Workaround ARM bug with stencil write mask
dc82ff15e Add config angle_maybe_has_histograms to angle_perftests_common.
17df2fd32 Roll vulkan-deps from fe14e7954b25 to c728ed48ce2f (8 revisions)
408c5a56b Mark FormatPrintTest as potentially uninstantiated.
ee0938386 Roll Chromium from 6306836d087b to d830698e47ae (506 revisions)
2c3513518 Vulkan: Don't break render pass on read-only buffer updates
1bb3685d0 Roll SwiftShader from 678ffefb1b3b to aafa10869568 (9 revisions)
8050079c1 Vulkan: Remove basicGLLineRasterization
52719c9f7 [fuchsia] Migrate angle_end2end_tests to component framework v2.
d8c6a2357 Roll vulkan-deps from 4b211a6a98e6 to fe14e7954b25 (6 revisions)
97b16799c Roll Chromium from 199441d23d93 to 6306836d087b (461 revisions)
6f80f0f03 Translator: Clean up the compile flag passing interface
8474a95e3 Translator: Don't validate structs on non-root nodes
7fe0694ce Vulkan: Use push constants for driver uniforms
c554b92b1 Apply memory qualifier decorations in SPIR-V
2cf9b1d63 Encapsulate prevTexture pointer in a class
52b18311b Use angle::HashSet to replace std::set in Display.h
f5de8e014 Vulkan: Disable texture upload if texture sharing
5b7b7f9b0 Document how to roll VMA for ANGLE
aaf293c89 Roll VK-GL-CTS from 0170b4a25e36 to eaf78cbfdaed (4 revisions)
2de8bb42a Converge run wrappers into angle_test_util.RunTestSuite
a2ad517f9 Replace std::vector with angle::FixedVector in FramebufferState.
22b90986c Roll vulkan-deps from fa6a3bcb6169 to 4b211a6a98e6 (8 revisions)
bf3955820 Roll Chromium from 5dde4351e359 to 199441d23d93 (45 revisions)
6491396c1 Use angle::FlatUnorderedMap as storage for AttributMap
ae0bb7416 Roll Chromium from 2bf3e35d7a4c to 5dde4351e359 (604 revisions)
6b248a688 Use angle::FastVector in ImageEGL::initialize to avoid heap allocation
0c8c4ca02 Remove not used mImageLastUseFences in ImageVk
0d5ba4dc4 Minor performance improvement
928c5016b Vulkan: Fix garbage collection vs outside-RP-only flush
d53086022 Test for ARM bug with dynamic stencil write mask
146329b81 run_perf_tests: Allow passing flags to the test.
9e3e20327 Add a note about "internal swarming failure"
123bd5422 Add Variable Initialization in Base Instance Tests
a46fdafd3 Roll vulkan-deps from 3a443029375b to fa6a3bcb6169 (5 revisions)
dbfab1595 Roll Chromium from d7560fcbdf63 to 2bf3e35d7a4c (416 revisions)
de5c6c791 Remove unused ANGLE histograms
6f158ca04 Roll vulkan-deps from 3ee9789f25ed to 3a443029375b (8 revisions)
861149c7a Make PLS coherent on desktop OpenGL
e6fcb2a9a Roll Chromium from 54b1e2bc5d71 to d7560fcbdf63 (301 revisions)
c0818a182 Protect ImageState::targets with a std::mutex
9f0cec145 dEQP: Fix X11 display config name.
c54ed7905 Get desktop GL conformance tests to build
448e2275f (Reland) [Metal] Disable ANGLE's Metal backend on older GPUs.
57ccd974b Make limit_max_texture_size_to_4096 webgl only
88b05d3c8 Use globals in android_helper to simplify flow.
553b1334e Vulkan: fix default msaa framebuffer resolve issue.
00a9787e3 Roll Chromium from fe6c14e1fc75 to 54b1e2bc5d71 (162 revisions)
8f14296cc Roll VMA to version 3.0.1 + Google patches
6bc106f74 Ignore all .gclient* files.
e88cab292 Revert "Skip multi draw indirect tests on Linux/Intel/Vulkan."
f0d2dbec7 Retry flaky failures on SwiftShader *SAN builds.
33c09ba9b Fix local gold runs broken by previous refactoring.
25a5a42df Roll vulkan-deps from 9501e63393f3 to 3ee9789f25ed (3 revisions)
0b7e347a6 Roll Chromium from 0920d63e6fcc to fe6c14e1fc75 (234 revisions)
05d86e060 Add temporary logging of gold_session.Initialize stdout.
47684b26f Roll Chromium from d268b4c8a6b1 to 0920d63e6fcc (256 revisions)
351a2a6b6 Skip multi draw indirect tests on Linux/Intel/Vulkan.
19c6600c4 Refactoring: move a few common functions to angle_test_util.
13da8b973 Tests: Add Octopath Traveler trace
c0f2a5ac8 Skip TimerQueriesTest flakes on NV/Linux.
245198f1e Skip flaky tests on Linux/Vulkan.
f62b4b940 Skip flaky image load/store tests on Win/D3D11.
c29100a8f Skip flaky test on SwiftShader.
6e6f3e6da Convert GLenum to string for failed trace prereq message
1353f1191 Roll vulkan-deps from 4818e22b9347 to 9501e63393f3 (1 revision)
8eaa0ccfc Revert "[Metal] Disable ANGLE's Metal backend on older GPUs."
0d0d9168c Roll SwiftShader from 67158b90f83c to 678ffefb1b3b (2 revisions)
60a02ca6e Roll Chromium from bfa66f9bea8c to d268b4c8a6b1 (202 revisions)
cc2095d25 [Metal] Disable ANGLE's Metal backend on older GPUs.
c517463ab Add NoUnclampedBlendColor limitation
81554b66a Vulkan: Reduce Framebuffer dirtyBits for swapchain only changes
49523e69d Roll Chromium from e810ac85e1c8 to bfa66f9bea8c (1114 revisions)
606a92395 Tests: Add Ni no Kuni trace
0b4ee2a8b Roll vulkan-deps from 73766817e984 to 4818e22b9347 (9 revisions)
6887c9505 Roll SwiftShader from 80aaa8a74b74 to 67158b90f83c (2 revisions)
dc77621bc Vulkan: Fix RefCountedDescriptorPoolHelper leak
d6acced1f Add flag --save-screenshots (implied by --screenshot-dir).

Canonical link: https://commits.webkit.org/253669@main
